andela/coding-geeks-ah-backend
# #167484575 configured authors haven to use postgresql as db engine and sequelize

#### What does this PR do?
It configures **Authors Haven** to use **PostgreSQL** as a database engine and **Sequelize** as **ORM** (Object Relational Mapping).

#### Description of Task to be done?
1. Setting up required dependencies/packages.
2. Configuring Sequelize to use the PostgreSQL database.
3. Testing the database connection by creating the server with Express.

#### How should this be manually tested?
1. Clone this branch.
2. Do **npm install**.
3. Create two databases called: **authors-haven** for development and **authors-haven-test** for testing.
4. Finally, Do **npm start** to check if you get a console message telling you that: **Database Connected!**
